src/main/resources/com/google/api/codegen/metadatagen/sample_manifest.snip

```diff
@@ -0,0 +1,26 @@
+# Template for sample manifest yaml file.
+
+@snippet generate(manifest)
+  ---
+  type: manifest/samples
+  schema_version: {@manifest.schemaVersion}
+  {@manifest.environment}: &{@manifest.environment}
+    environment: {@manifest.environment}
+    bin: {@manifest.bin}
+    base_path: {@manifest.basePath}
+    invocation: {@manifest.invocation}
```

Comment:
very tiny nit suggestion, totally not mandatory: consider having `invocation` after `package` for fewer back-references for human readers